src/docs/build-and-earn/testnet-guides/create-validator.md

```diff
@@ -5,16 +5,18 @@ title: Create a Validator
 # Create a validator
 
 To create a validator account, you need:
-1. A running ethereum full node
+1. A running ethereum full-node
 2. A validator
 3. An orchestrator
 4. A gravity bridge
 
 Only after finalizing previous steps, you can start the process of creating a validator and running it on your node. this section explains how to achieve each step in detail.
 
-## Ethereum full node
+## Ethereum full-node
 
-You can use an existing full-node if you have or you can follow the procedure below in order to start one but do use Infura:
+You need to use [ethereum full-node](https://ethereum.org/en/developers/docs/nodes-and-clients/#full-node).
+
+You can use an existing full-node if you have or you can follow the procedure below in order to start one but make sure not to use Infura:
```

Comment:
for clarity, maybe something like:

You can use an existing Ethereum full-node if you have one or you can follow the procedure below in order to start one but make sure ***not to use Infura***:

Reply:
steps 2, 3 and 4 below could be replaced with (similarly to https://cudos-network-docs.netlify.app/docs/build-and-earn/testnet-guides/prerequisites.html#setup-the-environment-before-running-a-node):
> git clone --depth 1 --branch sdk-0.43  https://github.com/CudoVentures/cudos-builders.git CudosBuilders

Reply:
similarly, step 5 and 6 (as in https://cudos-network-docs.netlify.app/docs/build-and-earn/testnet-guides/prerequisites.html#setup-the-environment-before-running-a-node + `ethereum.yml` does not exist!), could be replaced with single command:
> cd CudosBuilders/docker/ethereum && sudo docker-compose -f ethereum-full.yml -p ethereum up --build --detach

Reply:
@Nazeim have these issues been addressed in latest commits?